static/js/apps/nl_interface/app_state.ts

```diff
@@ -75,15 +75,58 @@ export interface NLQueryContextType {
 }
 
 interface NLAppConfig {
-  autoRun: boolean;
+  /**
+   * Auto play mode option: Automatically run the query; don't wait for the user to hit enter
+   * Enable with URL param: "a=True"
+   * Default: false
+   */
+  autoPlayAutoRunQuery: boolean;
+
+  /**
+   * Auto play mode option: Wait for the user to hit "enter" in the search box before showing the next query
+   * Enable with URL param: "m=True"
+   * Default: false
+   */
+  autoPlayManuallyShowQuery: boolean;
+
+  /**
+   * Auto play mode option: Wait this number of ms after a query completes before showing the next query
+   * Example: set with URL param: "qd=1000"
+   * Default: 5000 (5 seconds)
+   */
+  autoPlayAutoShowQueryDelay: number;
+
+  /**
+   * Auto play mode internal state: current query index
+   */
+  autoPlayCurrentQueryIndex: number;
+
+  /**
+   * Auto play mode option: Disable the typing animation on auto-play  mode
+   * Enable with URL param "d=True"
+   * Default: false
+   */
+  autoPlayDisableTypingAnimation: boolean;
```

Comment:
maybe name it autoPlayEnableTypingAnimation for easy understanding?

Reply:
leaving as is per offline conversation